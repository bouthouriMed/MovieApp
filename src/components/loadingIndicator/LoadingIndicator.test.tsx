import { render } from "@testing-library/react";
import Loading from "./LoadingIndicator";

describe("LoadingIndicator", () => {
  it("matches snapshot", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
