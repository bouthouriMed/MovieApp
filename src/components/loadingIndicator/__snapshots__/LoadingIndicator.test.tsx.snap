// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoadingIndicator > matches snapshot 1`] = `
<div>
  <div
    class="loading-indicator"
    data-testid="loading"
  >
    <div
      class="spinner"
    />
  </div>
</div>
`;
